src/Operation/Watch.php

```diff
@@ -79,22 +92,44 @@ class Watch implements Executable
      *  * resumeAfter (document): Specifies the logical starting point for the
      *    new change stream.
      *
+     *    Using this option in conjunction with "startAtOperationTime" will
+     *    result in a server error. The options are mutually exclusive.
+     *
      *  * session (MongoDB\Driver\Session): Client session.
      *
      *    Sessions are not supported for server versions < 3.6.
      *
+     *  * startAtOperationTime (MongoDB\BSON\TimestampInterface): If specified,
+     *    the change stream will only provide changes that occurred at or after
+     *    the specified timestamp. Any command run against the server will
+     *    return an operation time that can be used here. Alternatively, an
+     *    operation time may be obtained from MongoDB\Driver\Server::getInfo().
+     *
+     *    Using this option in conjunction with "resumeAfter" will result in a
+     *    server error. The options are mutually exclusive.
+     *
+     *    This option is not supported for server versions < 4.0.
+     *
      *  * typeMap (array): Type map for BSON deserialization. This will be
      *    applied to the returned Cursor (it is not sent to the server).
      *
-     * @param string         $databaseName   Database name
-     * @param string         $collectionName Collection name
+     * Note: A database-level change stream may be created by specifying null
+     * for the collection name. A cluster-level change stream may be created by
+     * specifying null for both the database and collection name.
+     *
+     * @param string|null    $databaseName   Database name
+     * @param string|null    $collectionName Collection name
      * @param array          $pipeline       List of pipeline operations
      * @param array          $options        Command options
      * @param Manager        $manager        Manager instance from the driver
```

Comment:
Why is "Manager" the last option in this list, but the first argument in the method declaration?

Reply:
Good catch! Looks like an old typo: https://github.com/mongodb/mongo-php-library/blame/master/src/Operation/Watch.php#L93

I'll fix it.